src/plugins/intel_gna/src/backend/gna_limitations.hpp

```diff
@@ -328,8 +341,12 @@ inline bool Limitations::is_aligned(size_t addr) const {
     return (addr == ALIGN(addr, get_memory_alignment()));
 }
 
+inline bool Limitations::has_internal_memory_buffers() const {
```

Comment:
It would be nice to have all getter functions for the wrapper class in one place.

Reply:
OK